executable/java_class.ksy

```diff
@@ -219,19 +293,49 @@ types:
         8: new_invoke_special
         9: invoke_interface
   method_type_cp_info:
-    doc-ref: 'https://docs.oracle.com/javase/specs/jvms/se8/html/jvms-4.html#jvms-4.4.9'
+    doc-ref: https://docs.oracle.com/javase/specs/jvms/se8/html/jvms-4.html#jvms-4.4.9
+    -orig-id: CONSTANT_MethodType_info
     seq:
+      - type: version_guard(51)
       - id: descriptor_index
         type: u2
+  dynamic_cp_info:
+    doc-ref: https://docs.oracle.com/javase/specs/jvms/se19/html/jvms-4.html#jvms-4.4.10
+    -orig-id: CONSTANT_Dynamic_info
+    seq:
+      - type: version_guard(55)
+      - id: bootstrap_method_attr_index
+        type: u2
+      - id: name_and_type_index
+        type: u2
   invoke_dynamic_cp_info:
-    doc-ref: 'https://docs.oracle.com/javase/specs/jvms/se8/html/jvms-4.html#jvms-4.4.10'
+    doc-ref: https://docs.oracle.com/javase/specs/jvms/se8/html/jvms-4.html#jvms-4.4.10
+    -orig-id: CONSTANT_Dynamic_info
     seq:
+      - type: version_guard(51)
       - id: bootstrap_method_attr_index
         type: u2
       - id: name_and_type_index
         type: u2
+  module_package_cp_info:
```

Comment:
This is now the only `*_cp_info` type without `-orig-id`.

I think you can list both:

```suggestion
  module_package_cp_info:
    -orig-id:
      - CONSTANT_Module_info
      - CONSTANT_Package_info
```